### PR TITLE
Show card resource icons instead of 'RES' in the corner.

### PR DESCRIPTION
**Pull request by kberg**
I don't like copying the CSS. I wonder if `less` can help here.

image

### CODE REVIEW

**Comment by kberg** [comment]
If the CSS copying can be resolved (or accepted as a copy, _eyyuch)_, this would be good to go.

**Comment by kevinb9n** [comment]
Any particular preference for "[Floater]: 5" over "5 [Floater]"?

**Comment by kberg** [comment]
I agree, swapping the order is nicer. Will do.

**Comment by kberg** [comment]
Done: 

image

File: src/styles/cards_v2.less [hunk 1]
```diff
@@ -1817,3 +1817,57 @@
...
+    .card-resource {
+        position: relative;
+        display: inline-block;
+        margin-bottom: 4px;
+        width: 22px;
+        height: 22px;
+        background-size: 22px;
+        vertical-align: middle;
+    }
+    .card-resource-microbe {
```
Comment (alrusdi) [inline src/styles/cards_v2.less:1832]: You can try to reduce code repetioton by using lesscss each keyword like this:
`@resource_types: microbe, animal, science;

each(@resource_types, {
    .card-resource-@{value} {
        background-image: url("./assets/resources/@{value}.png");
    }
})`

**Review by alrusdi — APPROVED** [review]

**Comment by kevinb9n** [comment]
I love this!

Have you checked that if one of these same cards like Security Fleet is on Self-Rep Robots that it doesn't accidentally show up as being fighter resources in that situation? It should look like blank white squares.

**Comment by kberg** [comment]
> I love this!
> 
> Have you checked that if one of these same cards like Security Fleet is on Self-Rep Robots that it doesn't accidentally show up as being fighter resources in that situation? It should look like blank white squares.

I have not checked SRR. I almost never play or test with promos. But it seems worth checking before applying.

**Comment by bafolts** [comment]
This is what gets displayed on the Self Replicating Robot cards.

image

This seems good enough for me for time being. It is at least not showing the fighter resource.

**Comment by kberg** [comment]
Yeah, this is looking decent: 

image

**Review by bafolts — APPROVED** [review]

**Review by kberg — COMMENTED** [review]
> This is what gets displayed on the Self Replicating Robot cards.
> 
> image
> 
> This seems good enough for me for time being. It is at least not showing the fighter resource.

Confirmed.

**Comment by kberg** [comment]
I removed the "prototype" from the definition because of satisfactory CSS reduction.

**Pull request merged**